games/games.js

```diff
@@ -83,28 +90,41 @@ logoutButton.addEventListener('click', () => {
  // on load . . .
 window.addEventListener('', async() => {
```

Comment:
Super close. This is not currently loading the past games when the /games page loads because you're missing the word load from inside the quotes on line 91. Just add that and you'll be in business!